### ISSUE TITLE
Testing out resnet with different LR schedulers

### ISSUE THREAD

**Issue by udaylunawat** [udaylunawat/Whats-this-rock#7]
## Notion page with resources
- [Learning rate schedulers](https://fantasy-bay-495.notion.site/Learning-rate-hyperparameter-tuning-6f2d84db96f640c9b46fe99b322c9048)

## LR Schedulers to try
- Steps
- Cosine with restarts
- Classic cosine annealing (per epoch)
- Classic cosine annealing (per batch)
- Classic cosine annealing (per batch) with warmup

## Inspiration Tweet
[Sebastian Raschka's Tweet](https://twitter.com/rasbt/status/1577429214755573761?s=21&t=Y5nST7iDXDs0mo61D_pDnw)
